### PR TITLE
Add GSAP Animation Library in animation.json

### LINKED ISSUE #1983: [Add] GSAP animation library
### Site URL

https://gsap.com/

### Description of the above provided site

**[GSAP](https://gsap.com/)** is a robust industry standard **JavaScript** animation library from GreenSock that lets you craft high-performance animations that work in every major browser. Animate CSS, SVG, canvas, React, Vue, WebGL, colors, strings, motion paths, generic objects...anything JavaScript can touch!

### Category

Frontend

### Sub Category

animations

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by psykat1116**
## Fixes Issue
Closes #1983 

## Changes proposed
Add New Animation Library called **GSAP** in the **frontend** category and **animations** subcategory.

### CODE REVIEW

**Review by CBID2 — APPROVED** [review]
Well done @psykat1116! :) @Anmol-Baranwal and @rupali-codes! :)

**Review by CBID2 — CHANGES_REQUESTED** [review]
Ignore my previous comment @psykat1116. It looks like the deployment failed. @rupali-codes, can you help figure out why?

File: database/frontend/animations.json [hunk 1]
```diff
@@ -131,5 +131,12 @@
...
     "category": "frontend",
     "subcategory": "animations"
+  },
+  {
+    "name": "GSAP",
+    "description": "GSAP is a robust industry standard JavaScript animation library from GreenSock that lets you craft high-performance animations that work in every major browser. Animate CSS, SVG, canvas, React, Vue, WebGL, colors, strings, motion paths, generic objects...anything JavaScript can touch!",
+    "url": "https://gsap.com/",
+    "catagory": "frontend",
+    "subcatagory": "animations"
   }
```
Comment (CBID2) [inline database/frontend/animations.json:141]: [proofreading] @psykat1116, I figured out why the deployment failed...formatting issues. Add a comma after the closing bracket.   
```suggestion
    "name": "GSAP",
    "description": "GSAP is a robust industry-standard JavaScript animation library from GreenSock that lets you craft high-performance animations that work in every major browser. Animate CSS, SVG, canvas, React, Vue, WebGL, colors, strings, motion paths, generic objects...anything JavaScript can touch!",
    "url": "https://gsap.com/",
    "category": "frontend",
    "subcategory": "animations"
  }, 
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Refer to my in-line comment @psykat1116.

**Comment by psykat1116** [comment]
@CBID2 I fix this issue please check is there any problem now.

**Comment by CBID2** [comment]
> @CBID2 I fix this issue please check is there any problem now.

The problem still remains @psykat1116.

**Comment by Anmol-Baranwal** [comment]
@psykat1116 
Deployment should not fail.

image

**Comment by Anmol-Baranwal** [comment]
@psykat1116 
Check json structure.

image

It is clearly mentioned in the contributing guidelines.

**Comment by psykat1116** [comment]
@Anmol-Baranwal there was a small typos. I fixed it now please tell me is there any problem.

**Review by CBID2 — DISMISSED** [review]
Looks good to me @Anmol-Baranwal

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**